bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -124,15 +121,17 @@ def detailed_perf_stats(config: WTPerfConfig, perf_stats: PerfStatCollection):
     return as_dict
 
 
-def run_test_wrapper(config: WTPerfConfig, operations: List[str] = None, arguments: List[str] = None):
+def run_test_wrapper(config: WTPerfConfig, index: int = 0, arguments: List[str] = None):
     for test_run in range(config.run_max):
         print("Starting test  {}".format(test_run))
-        run_test(config=config, test_run=test_run, operations=operations, arguments=arguments)
+        run_test(config=config, test_run=test_run, index=index, arguments=arguments)
         print("Completed test {}".format(test_run))
 
 
-def run_test(config: WTPerfConfig, test_run: int, operations: List[str] = None, arguments: List[str] = None):
-    test_home = create_test_home_path(home=config.home_dir, test_run=test_run, operations=operations)
+def run_test(config: WTPerfConfig, test_run: int, index: int = 0, arguments: List[str] = None):
+    test_home = create_test_home_path(home=config.home_dir, test_run=test_run, index=index)
+    if config.verbose:
+        print("Home directory created: {}".format(test_home))
```

Comment:
I think there's a chance  `Home directory created` might be cause someone to think that code is creating the directory. Perhaps changing to something like `Home directory path created`  would help.